nengo_gui/static/data_to_csv.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * A function that retuns simulation data as a csv
+ *
+ * @param {Nengo.Component.components} A list of the graph items in the simulation
+ */
+
+
+var data_to_csv = function(data_set){
+
+            var values = [];
+            var dim_values = [];
+            var times = [];
+            var csv = [];
+            var csv_string = "";
+
+            var data_set = data_set.filter(function(data){
+                return (data.constructor === Nengo.Value || data.constructor == Nengo.XYValue)
```

Comment:
They both should probably be ```===```, otherwise it looks like you're specifically doing things differently in the ```XYValue``` case